packages/common/README.md

```diff
@@ -4,30 +4,24 @@
 ￼
 This package exists to share code used by more than one data-driven-forms package, such as:
 
-* shared interfaces - propTypes
-* common mapper code - `condition` implementation
+- shared interfaces - propTypes
+- common mappers code 
+  - wizard
+  - select
+  - field array
```

Comment:
actually there is no common field-array yet

Reply:
You are right.